lib/cli/bin/generate.js

```diff
@@ -167,6 +167,13 @@ switch (projectType) {
       .catch(handleError);
     break;
 
+  case types.ANGULAR:
+    // eslint-disable-next-line
+    require('../generators/ANGULAR')
+      .then(commandLog('Adding storybook support to your "Angular" app'))
+      .then(end);
```

Comment:
You need to add `catch(handleError)` here

I’ll consider refactoring generate.js into async function with a single try/catch block, so that no promise rejection is left unhandled